src/main/app/Resources/styles/print.less

```diff
@@ -0,0 +1,100 @@
+@media print {
+    html {
+        height: auto !important;
+        overflow-y: auto;
+    }
+
+    .app-header-container {
+        display: none;
+    }
+
+    .breadcrumb {
+        display: none;
+    }
+
+    .page-actions {
+        display: none;
+    }
+
+    .tag-cell {
+        display: flex;
+        flex-flow: row wrap;
+    }
+
+    .app .app-menu {
+        display: none;
+    }
+
+    .app-content .app-footer-container {
+        display: none;
+    }
+
+    .overview-user-actions {
+        display: none;
+    }
+
+    .data-list .list-header {
+        display: none;
+    }
+
+    // subject to change, not sure how to select by class:
+    #ajouter-une-session-de-formati {
+        display: none;
+    }
+
+    .checkbox {
+        display: none;
+    }
+
+    .data-grid-item-select {
+        display: none;
+    }
+
+    .data-card-toolbar {
+        display: none;
+    }
+
+    .data-card-flags {
+        display: none;
+    }
+
+    .alert.alert-detailed.alert-success {
```

Comment:
Too restrictive selector. I think you'll want either `.alert` or `alert-detailed`. The later one will not hide all alerts in the app.